docs/source/usage/workflows/memoryPerAccelerator.rst

```diff
@@ -0,0 +1,115 @@
+.. _usage-workflows-memoryPerAccelerator:
```

Comment:
we (try to) call "accelerators" consequently "device" in PIConGPU 0.4+

Reply:
I think in this case calling CPUs "accelerators" would have been even more confusing.

Reply:
everything is a device now :)
(and we also program them as if all devices would be accelerators, muhahaha)